LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2024 NAME
+Copyright (c) 2025 NAME
```

Comment:
We can maybe replace this with the template text from [choosealicense](https://choosealicense.com/licenses/mit/) to allow the user to replace this when they follow the checklist list.
```suggestion
Copyright (c) [year] [fullname]
```